jenkins/pipelines/java/Jenkinsfile

```diff
@@ -17,17 +16,24 @@ pipeline {
                 }
             }
         }
-        stage('Run Tests') {
+
+        // Note that there is no fundamental reason that the Desktop and Web Services
+        // builds couldn't run in parallel.  We'd need to figure out how to set
+        // up the environment server and sg at alternate ports, probably, and we'd
+        // need hardware that can support the load.
```

Comment:
This note is exactly why I realized that ultimately making device specific locks is redundant.  Even if it's two different phones running servers, they will be talking to the same backend until we migrate to AWS.